config.c

```diff
@@ -0,0 +1,294 @@
+/**
+ * Config Reader and Parser
+ * This program will read the config for lightwm in the appdata directory
```

Comment:
No need for the description of what this file does in English words, this can be especially problamatic if someone changes the logic of the code and forgets to change the comment

I prefer to have in this case only the code as documentation :)